src/renderer/shared/i18n/locales/en.json

```
@@ -177,7 +177,7 @@
     "signatoryAddress": "Signatory address",
     "signatoryNameLabel": "Signatory name",
     "signatorySelection": "Choose or enter address",
-    "signatoryThresholdDescription": "Set the signatory wallet of your flexible multisig and how many need to confirm to execute a valid transaction",
+    "signatoryThresholdDescription": "Set the signatory wallet of your multisig and how many need to confirm to execute a valid transaction",
```

according to [layouts](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41410&t=T99kW8RMsCtXdMqz-1) the string is: "Step 2. Assign the signatories and set the signatory threshold for your multisig."

---

Damn I was looking at the wrong. Thank you 
```suggestion
    "signatoryThresholdDescription": "Assign the signatories and set the signatory threshold for your multisig.",
```